x/init.go

```diff
@@ -65,6 +65,10 @@ func Init() {
 	// TODO: why is this here?
 	// Config.QueryEdgeLimit = 1e6
 
+	if dgraphVersion == "" {
+		dgraphVersion = "dev"
```

Comment:
I don't know if it would be a conflict. I think we're doing the same thing, just for different purposes.

If there is a value. My change won't do anything. So DevVersion() won't break. It will add "dev" only for unforeseen cases.

The DevVersion() function expects some value to exist. This is probably a specific scenario. For example, "rc" releases. It's in a "dev" version. But my goal is to run main.go directly(other scenario). This function does not take this into account. And I wouldn't want to modify DevVersion(). Would have to add switch case logic. Unless you really think I should do this.

Also, this function just verify if it is a dev version release. It doesn't inject values in dgraphVersion variable. It is used just for sentry contextual purposes.

Again, same idea but totally different purposes.

Also, there is an internal old discussion about remove sentry completely. I think it was generating panics.

Cheers.